### PR TITLE
Prerequisites and Release notes for Tizen Studio 3.5 release

### PR DESCRIPTION
**Pull request by manishtr**
### Change Description ###

Prerequisites and Release notes for Tizen Studio 3.5 release

Signed-off-by: manish.r <manish.r@samsung.com>

### CODE REVIEW

**Comment by mijin85cho** [comment]
This update is expended version of #845.
So I assigned this PR to @Reni-90, the reviewer of #845.

@Reni-90 
Compared to #845, the 3-5-release-notes.md in this PR we have below additional lines

-  Support for adding dependent package information provided in config and manifest editor from Tizen 5.5.
 - Support for "api-version" attribute in application manifest from Tizen 5.5.
 - Support for "visibility" attribute in app-control element from Tizen 5.5.

- 32 bit systems support has been deprecated.

**Comment by Reni-90** [comment]
@mijin85cho 
Sure, I will.

File: docs/application/tizen-studio/release-notes/3-5-release-notes.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+### Fixed Bugs
+-   IDE
+	-   The issue of Samsung Certificate unable to sign application while launching an application on Tizen 5.0 Emulator has been fixed.
+
+### Deprecated
+-   IDE
+	-   Java 9 and OpenJDK 10 support has been deprecated.
+	-   Log Viewer has been deprecated.
+	-   32 bit systems support has been deprecated.
```
Comment (Reni-90) [inline docs/application/tizen-studio/release-notes/3-5-release-notes.md]: Please rephrase to:
32-bit system support has been deprecated.

Please verify.

**Comment by mijin85cho** [comment]
Thanks for the update and review.
I'll merge this PR near Oct 30, the actual release target date.

**Comment by kvbhat** [comment]
Hi @mijin85cho 
We moved this PR changes to https://github.com/Samsung/tizen-docs/pull/910 along with few updates as per review comments.
Can you please review and merge https://github.com/Samsung/tizen-docs/pull/910 ?

**Comment by mijin85cho** [comment]
Hello, @kvbhat 
Is there any reason that we can't update this PR #890 and move to #910?
As it is not merged yet, we can update this PR.

**Comment by mijin85cho** [comment]
Comparing to #910, there only removal for some sections.
So I think we can skip the review even after updating this PR.

**Comment by kvbhat** [comment]
@journey2w Thanks the PR looks good to me. The new changes are part of this PR now.
Thanks again.

**Pull request merged**